tests/phpunit/CRM/Event/Form/Registration/ConfirmTest.php

```diff
@@ -100,6 +100,7 @@ public function testSubmit() {
    */
   public function testPaidSubmit($thousandSeparator) {
     $this->setCurrencySeparators($thousandSeparator);
+    $mut = new CiviMailUtils($this);
```

Comment:
@eileenmcnaughton I've not seen this before - is it a new feature in the tests? Or just something we've not used much?

Reply:
@mattwire actually neither  it's been around a while & we use it quite a  lot :-)